### PR TITLE
feat(tracing-apps): add jaeger-operator

### PR DESCRIPTION
**Pull request by hairmare**

### CODE REVIEW

**Comment by eyenx** [comment]
Not sure if it's the right module to package this in `logging-apps` 

Why not `tracing-apps`

**Comment by eyenx** [comment]
Thoughts @tongpu

File: charts/logging-apps/Chart.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ name: logging-apps
 description: Argo CD app-of-apps config for logging applications
 type: application
 # version and appVersion are in sync in this chart!
-version: 0.0.2
-appVersion: 0.0.1
+version: 0.1.0
```
Comment (eyenx) [inline charts/logging-apps/Chart.yaml:6]: @tongpu and I were discussing this last week. 

Do we need to do a minor release bump when ADDING a new app? Or would a patchversion bump suffice? What are your thoughts @hairmare ?

Comment (hairmare) [reply charts/logging-apps/Chart.yaml:6]: As per semver, bumping the minor version indicates that a new feature was added. A patch bump would indicate that we are fixing an issue.

**Comment by eyenx** [comment]
no example needed in `charts/logging-apps/examples/jaegerOperator.yaml` ?

File: charts/logging-apps/Chart.yaml (hunk 1 above)
Comment (eyenx) [reply charts/logging-apps/Chart.yaml:6]: Okay this makes totally sense.

**Comment by hairmare** [comment]
> no example needed in `charts/logging-apps/examples/jaegerOperator.yaml` ?

none available since I haven't really played with it to figure out how a complete example would look.

**Comment by hairmare** [comment]
> Not sure if it's the right module to package this in `logging-apps`
> 
> Why not `tracing-apps`

Maybe we should rename `logging-apps` to `instrumentation-apps`... I'm a bit wary of us creating a new `-apps` chart for each new thing but am fine with either solution.

**Comment by eyenx** [comment]
> > Not sure if it's the right module to package this in `logging-apps`
> > Why not `tracing-apps`
> 
> Maybe we should rename `logging-apps` to `instrumentation-apps`... I'm a bit wary of us creating a new `-apps` chart for each new thing but am fine with either solution.

`observability-apps` ?

**Comment by hairmare** [comment]
> > > Not sure if it's the right module to package this in `logging-apps`
> > > Why not `tracing-apps`
> > 
> > 
> > Maybe we should rename `logging-apps` to `instrumentation-apps`... I'm a bit wary of us creating a new `-apps` chart for each new thing but am fine with either solution.
> 
> `observability-apps` ?

`o11y-apps` :blush:

**Comment by hairmare** [comment]
If we rename it to observability we would also need to consider putting monitoring stuff into the same `-apps`. I'll most likely end up doing a new `tracing-apps` but I'm trying to think of some others things that would fit into the same chart as well before I do so.

**Comment by tongpu** [comment]
> > > > Not sure if it's the right module to package this in `logging-apps`
> > > > Why not `tracing-apps`
> > > 
> > > 
> > > Maybe we should rename `logging-apps` to `instrumentation-apps`... I'm a bit wary of us creating a new `-apps` chart for each new thing but am fine with either solution.
> > 
> > 
> > `observability-apps` ?
> 
> `o11y-apps`

I am Lukas Grossar and I approve this message.

File: charts/logging-apps/values.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ filebeat:
 # fluentd -- [fluentd](https://github.com/fluent/fluentd) ([example](./examples/fluentd.yaml))
 # @default -- -
 fluentd:
-  # fluentd.enabled -- Enable loki
+  # fluentd.enabled -- Enable fluentd
```
Comment (tongpu) [inline charts/logging-apps/values.yaml:44]: I guess fixing this typo was introduced by `helm-docs`?

**Review by tongpu — APPROVED** [review]
LGTM

File: charts/logging-apps/values.yaml (hunk 2 above)
Comment (hairmare) [reply charts/logging-apps/values.yaml:44]: It was probably copy-pasta from when we added fluentd.

**Pull request merged**